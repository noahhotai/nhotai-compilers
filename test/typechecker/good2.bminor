


main: function integer () = {

    if (true){
        x: string = "dfsdfs";
    }

}