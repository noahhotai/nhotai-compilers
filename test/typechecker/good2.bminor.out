Program parsed successfully.
Program resolved successfully.
Program type checked successfully.
